geant3.sh

```diff
@@ -1,8 +1,9 @@
 package: GEANT3
 version: "%(tag_basename)s"
-tag: v3-7
+tag: v3-9
 requires:
   - ROOT
```

Comment:
We can remove ROOT as VMC already depends on ROOT

Reply:
removed